virt/virtual_machines/advanced_vm_management/virt-assigning-compute-resources.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-assigning-compute-resources"]
+= Assigning compute resources
+include::_attributes/common-attributes.adoc[]
+:context: virt-assigning-compute-resources
+
+toc::[]
+
+In {VirtProductName}, compute resources assigned to virtual machines (VMs) are backed by either guaranteed CPUs or time-sliced CPU shares.
+
+Guaranteed CPUs, also known as CPU reservation, dedicate CPU cores or threads to a specific workload, which makes them unavailable to any other workload. Assigning guaranteed CPUs to a VM ensures that the VM will have sole access to a reserved physical CPU. xref:../../../virt/virtual_machines/advanced_vm_management/virt-dedicated-resources-vm.adoc#virt-dedicated-resources-vm[Enable dedicated resources for VMs] to use a guaranteed CPU.
+
+Time-sliced CPUs dedicate a slice of time on a shared physical CPU to each workload. You can specify the size of the slice during VM creation, or when the VM is offline. By default, each vCPU receives 100 milli, or 1/10 of a second, of physical CPU time.
```

Comment:
"milli" correct? Specify "millisecond"?

Reply:
@fabiand or @kbidarkar Please confirm if this should be milli or milliseconds.

Reply:
milliseconds is fine.